protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -845,9 +845,9 @@ func genEnumValidator(g *protogen.GeneratedFile, f *protogen.Field, varName stri
 	g.P("}")
 }
 
-func printErrorString(g *protogen.GeneratedFile, varName, fieldName, specificErr string) {
+func printErrorString(g *protogen.GeneratedFile, varName, fieldName, specificErr string, maxLen int) {
 	// Do not reflect untrusted value in error, certainly not for sensitive fields like password or PII like email
-	g.P(fmtPackage.Ident("Println"), "(`[log-only] ", fieldName, `: value must `, specificErr, "`)")
+	g.P(fmtPackage.Ident("Printf"), `("[log-only] %s: value must %s: %s\n", `, `"`, fieldName, `", "`, specificErr, `" `, ", proto.FirstCharactersFromString(", varName, ", ", maxLen, "))")
```

Comment:
We should normally not log the unvalidated data that didn't satisfy the validation rules. I can see why you want that in this instance but we have to sanitise the data before writing it to log. Can you please add a sanitise pattern that strips out anything other than alpha, numeric and the few additional characters that you expect here (`-` and `_`?)?

Reply:
makes sense.
I will use base64 on the resulting trimmed string